diff --git a/gnu.py b/gnu.py
index 49df885..c0694a6 100644
--- a/gnu.py
+++ b/gnu.py
@@ -328,7 +328,7 @@ class Gnu95FCompiler(GnuFCompiler):
                 if c_compiler and c_compiler.compiler_type == "msvc":
                     return []
                 else:
-                    raise NotImplementedError("Only MS compiler supported with gfortran on win64")
+                    pass #raise NotImplementedError("Only MS compiler supported with gfortran on win64")
         return opt
 
     def get_target(self):
